wandb/sdk/launch/_project_spec.py

```diff
@@ -33,8 +33,9 @@ def __init__(
     ):
 
         self.uri = uri
-        self.name = name  # todo: what to do for default names
+        self.name = name
         if self.name is None and utils._is_wandb_uri(uri):
+            # default name is {project}_{runid}
```

Comment:
since we're naming the run after the old run_id, maybe we should add a launch prefix?